app/Http/Resources/V1/H5pActivityResource.php

```diff
@@ -23,7 +23,12 @@ public function __construct($resource, $h5p_data)
      * @return array
      */
     public function toArray($request)
-    {        
+    {    
+          
+        //set playlist with project relation
+        $playlist = $this->playlist;
+        $playlist->project = $this->playlist->project;
```

Comment:
https://laravel.com/docs/7.x/eloquent-resources#writing-resources
Perhaps, we could use the resource for Playlist, if that already includes projects in response.
Just adding that as a note for now.